jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalAuthenticator.java

```diff
@@ -446,5 +451,36 @@ public JCUser storeRegisteredUser(UserDto userDto) {
         LOGGER.info("JCUser registered: {}", user.getUsername());
         return user;
     }
-    
+
+    private void removePasswords(final RegisterUserDto registerUserDto) {
```

Comment:
Please add JavaDocs explaining what this does and why it's needed.

Also, we don't have a habit to put `final` for local vars and parameters :)